openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -340,3 +341,10 @@ def pre_requisite_courses(self):
         Returns a list of ID strings for this course's prerequisite courses.
         """
         return json.loads(self._pre_requisite_courses_json)
+
+    @classmethod
+    def get_all_courses(cls):
+        """
+        Returns all courses from course overviews.
+        """
+        return [course_overview.id for course_overview in cls.objects.all()]
```

Comment:
Do you need this to return all course_overviews, or just all course-ids.  If the latter, we should update the method name to get_all_course_keys, or the like.